demo/src/app/components/typeahead/overview/sections/user-interaction/template.html

```diff
@@ -0,0 +1,82 @@
+<h5>Edit the search term</h5>
+
+<p>
+  The classic usage of the Typeahead is to use the input to edit the search term, and trigger a search process which will spawn a new options list.
+</p>
+
+<p>
+  Apart from creating an Observable based on the <code>input</code> event of the input, the Typeahead does nothing else specific, so to edit the search term just use it as any other input.
+</p>
+
+
+
+<h5>Navigate through options in the popup</h5>
```

Comment:
Also, this would be more than enough to list ONLY keyboard shourcuts as a simple table like here → https://ng-bootstrap.github.io/#/components/datepicker/overview#keyboard-shortcuts

Reply:
I disagree with that. If we are talking about user interactions, a lot of people are using the mouse, and Typeahead itself implements things specifically for the mouse. So removing that would be providing half the information.

Regarding the "1 table" vs "multiple tables", I just wanted to separate by intent. As a user, I will first think about doing something specific (navigate, select, close, etc.) and then will want to know how I can do it (keyboard, mouse, API, etc.). So the sectioning totally makes sense. And it is not breaking the readability. I could add some specific styling to visually separate the sections a bit more though.

Reply:
"Seeing" is entirely linked to the extent to which the user plays with the widget, and we can't know what they are going to do. So the statement "you see" can't be proved.

It is the same for "you know these things", we can't just consider what people know or not when it directly touches this specific implementation we provide. We can for pre-requisites such as Angular or RxJS, etc.

If I go with your logic, I perfectly know that <kbd>escape</kbd> will close all typeaheads, so no need to tell it either! Same for using <kbd>enter</kbd> and <kbd>space</kbd>. We can't have double standards.

You example with the button is not appropriate here. You are describing the behavior of a standard button on the web platform. We can consider that as an implicit pre-requisite and not describe it again.

But our typeahead implementation differs from the standard `select` element or any other similar widgets I've seen in any other application. There is a reason I described those behaviors, and that's because they belong to the minority as far as I personally know.

To quickly prove my point, pick a similar widget (standard `select`, a browser URL bar, a website with a search field) a,d try the following scenario: 

- make the popup open
- navigate with the keyboard
- hover another item with the mouse: did the highlighting change or are there two different elements highlighted?
- navigate again with the keyboard: which item became highlighted then? The one before/after the one hovered the mouse or the one before/after the one previously highlighted with he keyboard?

If we can consider a standard typeahead, our implementation is not matching it, hence a comprehensive documentation. And back to "it's obvious", "we know", you are yourself telling to describe things which can fall into the "obvious" category, so I don't see any partial limit here.